www/js/banidb/realm-search.js

```diff
@@ -1,7 +1,29 @@
+/* eslint-disable import/no-dynamic-require, global-require */
+const electron = require('electron');
+const path = require('path');
 const Realm = require('realm');
-const realmDB = require('./realm-db');
+
 const CONSTS = require('./constants');
 
+const { remote } = electron;
+const userDataPath = remote.app.getPath('userData');
+const realmPath = path.resolve(userDataPath, 'sttmdesktop-evergreen.realm');
```

Comment:
just wondering, is the new one intentionally sparse at the moment? not seeing any translations after checking out this branch.

Reply:
It is intentionally sparse, yes. Unicode and Transliterations were removed and will be done on-the-fly if needed with `anvaad-js`. Translations should be there, but all combined as a JSON object in the `Translations` column

Reply:
I see. Looks like we need to update some code use display it properly.